docs/components/filter-new.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: layouts/component.njk
+title: Filter
+---
+
+Use the filter component to let users filter a large list of items.
+
+#### Contributors
```

Comment:
This should be an h2, otherwise the title order won't make sense to a screenreader
```suggestion
## Contributors
```

Similarly, all the other headers in this file need to be increased by a level (e.g. `# Contents` to `## Contents`, `### Date picker` to `#### Date picker`) as there's already an h1 on this page for the page title ("Filter")

Reply:
Done